### DISCUSSION TITLE
is it possible to add openid4vc-verifier display metadata ?

### DISCUSSION THREAD

**Discussion by Sshovon** [openwallet-foundation/credo-ts#2063] [Q&A]
When creating an openid4vc-issuer using the `OpenId4VciCreateIssuerOptions`, issuer display metadata (such as label and image) can be included to enhance visibility on the wallet or holder interface. To further improve user experience and transparency, it would be beneficial to introduce verifier metadata during the openid4vc-verifier creation process. This verifier metadata would allow verifiers to provide their own display information, like labels and images, enabling the presentation layer to clearly show verifier details during credential presentations.

**Comment by TimoGlastra** [comment]
This is now supported in latest 0.6 alpha
